exercises/_template.livemd

```diff
@@ -0,0 +1,54 @@
+# EXERCISE-NAME
+
+```elixir
+Mix.install([
+  {:youtube, github: "brooklinjazz/youtube"},
+  {:hidden_cell, github: "brooklinjazz/hidden_cell"},
+  {:tested_cell, github: "brooklinjazz/tested_cell"},
+  {:utils, path: "#{__DIR__}/../utils"}
+])
+```
+
+## Navigation
+
+[Return Home](../start.livemd)<span style="padding: 0 30px"></span>
+[Report An Issue](https://github.com/DockYard-Academy/beta_curriculum/issues/new?assignees=&labels=&template=issue.md&title=)
+
+## EXERCISE-NAME
+
+EXERCISE-DESCRIPTION
+
+## Your Turn
```

Comment:
I think we can remove this Your Turn section from the template. I usually reserve Your Turns for the reading material.

I'd recommend:

## EXERCISE-NAME

EXERCISE-DESCRIPTION

<details style="background-color: lightgreen; padding: 1rem; border-radius: 0.5rem; margin: 1rem 0;">
<summary>Example Solution</summary>

```elixir
defmodule EXERCISE-NAME do
  def XXX() do
  end
end
```
</details>

Implement the `EXERCISE-NAME` module as documented below.
```

```elixir
defmodule EXERCISE-NAME do
  def XXX() do
  end
end
```

Reply:
ugh .. added comment and clicked resolve didnt save comment:

done!